packaging/nnstreamer.spec

```diff
@@ -57,6 +57,11 @@
 %define		vivante_support 0
 %endif
 
+# Disable NNFW if it's not ARM/x64 and x86
+%ifnarch %arm aarch64 x86_64 i586 i686 %ix86
```

Comment:
This is always FALSE in Tizen. We do not add %if for this condition :)
Let's make errors if a new arch (e.g., RISC-V) is introduced.

Reply:
Right. I will fix it.